modules/nw-networkpolicy-create-cli.adoc

```diff
@@ -116,6 +117,127 @@ spec:
 ifdef::multi[]
 +
 --
+
+where
+
+`<network_name>`:: Specifies the name of a network attachment definition.
+--
+endif::multi[]
++
+.Allow ingress traffic to one pod from a particular namespace
++
+This policy allows traffic to pods labelled `pod-a` from pods running in `namespace-y`.
++
+[source,yaml]
+----
+ifndef::multi[]
+kind: NetworkPolicy
+apiVersion: networking.k8s.io/v1
+endif::multi[]
+ifdef::multi[]
+apiVersion: k8s.cni.cncf.io/v1beta1
+kind: MultiNetworkPolicy
+endif::multi[]
+metadata:
+  name: allow-traffic-pod
+ifdef::multi[]
+  annotations:
+    k8s.v1.cni.cncf.io/policy-for: <network_name>
+endif::multi[]
+spec:
+  podSelector:
+   matchLabels:
+      pod: pod-a
+  policyTypes:
+  - Ingress
+  ingress:
+  - from:
+    - namespaceSelector:
+        matchLabels:
+           kubernetes.io/metadata.name: namespace-y
+----
+ifdef::multi[]
++
+--
+
+where
+
+`<network_name>`:: Specifies the name of a network attachment definition.
+--
+endif::multi[]
++
+ifdef::multi[]
+.Restrict traffic to a service
++
+This policy when applied ensures every pod with both labels `app=bookstore` and `role=api` can only be accessed by pods with label `app=bookstore`. In this example the application could be a REST API server, marked with labels `app=bookstore` and `role=api`.
++
+This example addresses the following use cases:
+
+* Restricting the traffic to a service to only the other microservices that need to use it.
+* Restricting the connections to a database to only permit the application using it.
++
+[source,yaml]
+----
+ifndef::multi[]
+kind: NetworkPolicy
+apiVersion: networking.k8s.io/v1
+endif::multi[]
+ifdef::multi[]
+apiVersion: k8s.cni.cncf.io/v1beta1
+kind: MultiNetworkPolicy
+endif::multi[]
+metadata:
+  name: api-allow
+ifdef::multi[]
+  annotations:
+    k8s.v1.cni.cncf.io/policy-for: <network_name>
+endif::multi[]
+spec:
+  podSelector:
+    matchLabels:
+      app: bookstore
+      role: api
+  ingress:
+  - from:
+      - podSelector:
+          matchLabels:
+            app: bookstore
+----
+ifdef::multi[]
++
+--
+
+where
+
+`<network_name>`:: Specifies the name of a network attachment definition.
+--
+endif::multi[]
+endif::multi[]
++
+ifdef::multi[]
+.Allow traffic from external clients
++
+This policy allows external service from the public Internet directly or through a Load Balancer to access the pod. Traffic from an external service is only allowed to a pod with the label `app=web`.
```

Comment:
Sorry, also this phrase seems to be not 100% coherent with the example. Pods with `app=web` can be accessed by everyone (inside and outside the cluster). In addition, an external service can connect to other pods as well. Honestly, I don't know how to implement the scenario you describe. Do you have some reference with regular NetworkPolicies doing that?

Reply:
Oh, great. With the "deny-all" policy in place, the statement makes a lot of sense

Reply:
@zeeke I updated with a worked example end-to-end https://51793--docspreview.netlify.app/openshift-enterprise/latest/networking/multiple_networks/configuring-multi-network-policy.html#nw-networkpolicy-allow-external-clients_configuring-multi-network-policy. To be honest, I tried to configure this without much success. I think it would need a QE in case I made a error. UI would suggest just install the allow policy it's one of the canned samples. PTAL